sass/includes/_record-links.scss

```
@@ -0,0 +1,92 @@
+.record-links {
+    &__wrapper {
+        display: flex;
+        align-items: center;
+        flex-direction: column;
+        padding-left:10%;
+        padding-right:10%;
+
+        @media (max-width: $screen__md) {
+            padding-left:5%;
+            padding-right:5%;
+        }
+    }
+
+    &__list {
+        list-style: none;
+        margin-bottom: 0px;
+        width: 100%;
+        padding-left: 0;
+        border-top: 0.3rem solid #ffcc00;
+    }
+
+    &__list a {
+        text-decoration: none;
+        color: inherit;
+    }
+    
+    &__list a:hover &__item,
+    &__list a:focus &__item{
+        outline: 0.312rem solid $color__dark;
+    }
+
+    &__item {
+        display: flex;
+        align-items: center;
+        border: 1px solid #D9D9D6;
```

I added a new variable for this colour in a recent PR - looks like it's `$color__grey-9` :)

---

```suggestion
        border: 1px solid #D9D9D6;
        border-top: 0;
```

We can get rid of the border-top - this will eliminate the 'double border' effect when two items sit together. The parent container's top yellow border acts as the top border for the first item in the list, so we can remove the top border for the individual items altogether :)